src/components/input-group/input-group.scss

```
@@ -0,0 +1,28 @@
+@use "../../compiled/tokens/scss/size";
+
+.c-input-group {
+  display: flex;
+
+  & :first-child {
+    border-bottom-right-radius: var(--size-border-radius-input-group);
+    border-top-right-radius: var(--size-border-radius-input-group);
+  }
+
+  & :last-child {
+    border-bottom-left-radius: var(--size-border-radius-input-group);
+    border-top-left-radius: var(--size-border-radius-input-group);
+  }
```

Should probably constrain selectors to immediate children...

```suggestion
  & > :first-child {
    border-bottom-right-radius: var(--size-border-radius-input-group);
    border-top-right-radius: var(--size-border-radius-input-group);
  }

  & > :last-child {
    border-bottom-left-radius: var(--size-border-radius-input-group);
    border-top-left-radius: var(--size-border-radius-input-group);
  }
```